hpx/parallel/algorithms/search.hpp

```diff
@@ -66,7 +66,7 @@ namespace hpx {namespace parallel { HPX_INLINE_NAMESPACE(v1)
 
                 s_difference_type diff = std::distance(s_first, s_last);
                 if (diff <= 0)
-                    return result::get(std::move(last));
+		  return result::get(std::move(first));
```

Comment:
I think this messed up the indentation. Please make sure that your editor is not inserting tabs into the files (see: https://github.com/STEllAR-GROUP/hpx/wiki/HPX-Source-Code-Structure for general coding guidelines).